website/docs/blueprints/distributed-databases/trino.md

```diff
@@ -94,26 +94,19 @@ kubectl get pods --namespace=trino
 ```bash
 NAME                                 READY   STATUS    RESTARTS   AGE
 trino-coordinator-5cfd685c8f-mchff   1/1     Running   0          37m
-trino-worker-6c896d96d6-kr895        1/1     Running   0          37m
-trino-worker-6c896d96d6-llp9j        1/1     Running   0          37m
-trino-worker-6c896d96d6-zpglv        1/1     Running   0          37m
 ```
-Next, we will retrieve the Application Load Balancer DNS fronting Trino service from Ingress object
+
+Next, we will port-forward the trino service so it can be accessed locally
 
 ```bash
-export TRINO_UI_DNS=$(kubectl describe ingress --namespace=trino | grep Address: | awk '{ print "http://"$2 }')
-echo $TRINO_UI_DNS
-```
-#### Output
-```bash
-http://k8s-trino-trinocoo-f64c9587b5-1488329710.us-west-2.elb.amazonaws.com
+kubectl -n trino port-forward service/trino 8080:8080
 ```
 
-Now, lets access the Trino UI by pasting above Load Balancer DNS in a web browser and login with username `admin` in the login window as shown below:
+Now, lets access the Trino UI at `http://localhost:8080` through web browser and login with username `admin` in the login window as shown below:
 
 ![Trino UI Login](./img/trino-ui-login.PNG)
 
```

Comment:
Please replace the Trino UI screenshot with "0" Active Workers to match the initial 0 active worker configuration above.